webanno/webanno
# #1295 Open CurationPage for specific document

For the development of resuming previous work in documents, I would like to open the CurationPage on a specific document via PageParameters passed to the constructor. Currently, this throws NPEs. I would also like to disable the Open-Document-Dialog for this case.

**How to test manually**
* checkout inception https://github.com/inception-project/inception/pull/1142
* start curation on a document 
* got to the project dashboard, select the document from the activities list
-> should open this document on the curation page
